src/sections/FourthSection.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+
+import FeaturesNavButton from '../components/FeaturesNavButton'
+import reportTruck from '../images/report-truck.jpg'
+import womenInTruck from '../images/women-in-truck.jpg'
+import deliveredPackage from '../images/package.jpg'
+import warehouse from '../images/warehouse.jpg'
+
+import { RiLineChartLine, RiFileChartLine, RiSurveyLine, RiFileExcelLine } from 'react-icons/ri' 
+
+import '../styles/fourthSection.css';
+
+
+const FourthSection = () => {
+    const [currentFeature, setCurrentFeature] = useState<string>('Dashboard')
+
+    const featuresNavButtons = [
+        {
+            label: 'Dashboard',
+            icon: <RiLineChartLine />
```

Comment:
icon: <RiLineChartLine /> is this done on purpose that this component is repeated twice?  (in reports?)

Reply:
Hahah it's not the same xD